silkrpc/txpool/transaction_pool.hpp

```diff
@@ -42,9 +43,26 @@
 namespace silkrpc::txpool {
 
 struct StatusInfo {
-   unsigned int queued;
-   unsigned int pending;
-   unsigned int base_fee;
+    unsigned int queued;
+    unsigned int pending;
+    unsigned int base_fee;
+};
+
+enum Type {
+    QUEUED,
+    PENDING,
+    BASE_FEE
+};
+
+struct ElementInfo {
+    Type type;
+    evmc::address sender;
+    silkworm::Bytes rlp;
+};
+
+
+struct TxPoolTransactions {
+    std::vector<ElementInfo> txs;
 };
```

Comment:
I suggest another name change here: `TxPoolTransactions` -> `TransactionsInPool`

Moreover, this `struct` could be replaced by: `using TransactionsInPool = std::vector<TransactionInfo>;`

Reply:
ok